src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -649,7 +673,7 @@ std::vector<std::string> Plugin::get_device_capabilities(const cldnn::device_inf
 uint32_t Plugin::get_max_batch_size(const ov::AnyMap& options) const {
     auto device_id = get_property(ov::device::id.name(), options).as<std::string>();
     auto context = get_default_contexts().at(device_id);
```

Comment:
Sees that we only need get_default_context(device_id, true); here too?

Reply:
I didn't explicitly initialize the context here to avoid early context and device initialization in cases where this property is requested for an empty ov::Model - so we can keep the device and context uninitialized